TheMovieDb/Scenes/Detail/Adapter/DetailDataSource.swift

```diff
@@ -20,11 +20,13 @@ protocol DataSourceProtocol {
 
 final class DetailDataSource: DataSourceProtocol {
     var items: [AnyHashable] = []
+    weak var delegate: DetailReviewsCellDelegate?
```

Comment:
This could be an implicitly unwrapped optional since the init requires this object.